embedchain/embedder/base.py

```diff
@@ -3,12 +3,20 @@
 from embedchain.config.embedder.base import BaseEmbedderConfig
 
 try:
-    from chromadb.api.types import Documents, Embeddings
+    from chromadb.api.types import Embeddings, Embeddable, EmbeddingFunction
 except RuntimeError:
     from embedchain.utils import use_pysqlite3
 
     use_pysqlite3()
-    from chromadb.api.types import Documents, Embeddings
+    from chromadb.api.types import Embeddings, Embeddable, EmbeddingFunction
```

Comment:
Let's remove this try except block. I think this is not needed. Might want to check though in a linux based docker container.